src/backend/distributed/executor/partitioned_intermediate_results.c

```diff
@@ -266,7 +266,8 @@ StartPortalForQueryExecution(const char *queryString)
 	/* don't display the portal in pg_cursors, it is for internal use only */
 	portal->visible = false;
 
-	PortalDefineQuerySelectCompat(portal, NULL, queryString, list_make1(queryPlan), NULL);
+	PortalDefineQuery(portal, NULL, queryString, CMDTAG_SELECT_COMPAT, list_make1(
+						  queryPlan), NULL);
```

Comment:
nit; try to keep the list statement on 1 line. probably break before `list_make1`. This makes it easier to see the list is 1 argument and keeps the parentheses that open and close more aligned.